subgraphs/orbit/src/common/constants.ts

```diff
@@ -0,0 +1,177 @@
+import { BigDecimal, BigInt } from "@graphprotocol/graph-ts";
+
+////////////////////
+///// Versions /////
+////////////////////
+
+export const PROTOCOL_NAME = "Orbit Bridge";
+export const PROTOCOL_SLUG = "Orbit Bridge";
```

Comment:
These are the only 2 lines you need to keep in this file

Reply:
Done